package.json

```diff
@@ -46,7 +47,8 @@
     "run-example": "cd example && npm start",
     "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
-    "react-scripts": "react-scripts"
+    "react-scripts": "react-scripts",
+    "live-edit": "concurrently \"npm run start &\" \"cd example && npm run start\""
```

Comment:
Okay so I don't want to bikeshed too much but with all the "build-blah-and-blah-blah" should this be something like `build-all-and-watch`

Reply:
This is actually a question -- not necessarily definitive. Maybe casually poll some folks and see what the general feeling is?

Reply:
Sounds good! I was hoping for some input on the name.

Reply:
I'm trying to think of a better name without too much wording as well, but if we want this to follow the other name schemes maybe we name it something like `run-example-and-watch`?